manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -218,8 +218,8 @@ nmi:
   findIdentityRetryIntervalInSeconds: ""
 
   # Enable scale features - https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.featureflags.md#enable-scale-features-flag
-  # Accepted values are true/false. Default is false.
-  enableScaleFeatures: ""
+  # Accepted values are true/false. Default is true for v1.8.1+.
+  enableScaleFeatures: true
```

Comment:
ditto

Reply:
The default is true but user's can still disable if they want to. We can remove this configuration later if we decide to not support disabling it. WDYT?

Reply:
Sounds good